src/reusecore/Button/btn.style.js

```diff
@@ -40,6 +40,12 @@ const ButtonStyle = styled.button`
             background: ${props.theme.highlightColor}; 
             box-shadow: 0 2px 10px ${props.theme.DarkTheme ? "rgb(255 255 255 / 40%)" : "rgb(0 0 0 / 40%)"};
         }
+
+        &:focus{
+            color: ${props.theme.black};
+            background: ${props.theme.highlightColor}; 
+            box-shadow: 0 2px 10px ${props.theme.DarkTheme ? "rgb(255 255 255 / 40%)" : "rgb(0 0 0 / 40%)"};
+        }
```

Comment:
Instead, update the value at line:22 and use `selectionColor` for background and `white` for color.
Also, can you remove the reference of `props.active` from the code while you are there? We don't have/use any prop named active in the component.

Reply:
@Nikhil-Ladha sorry for the late reply I was having fever. So, when I am updating at line 22 the other button such as subscribe button's colour is also getting changed.

Reply:
Yeah, both buttons should have the same active color. It's fine

Reply:
Yeah, we want all the buttons to have the same :active color.

Reply:
@Nikhil-Ladha  I have made the changes but I am not able to understand part of "use selectionColor for background" I googled it is showing selectionColor is used in reactnative for text input. Is selectionColor some short of themestyle. Can you please help.

Reply:
Well, it's a themeStyle variable.
But, it would be better to add a `activeColor` variable with the same value as selectionColor variable.

Reply:
Done @Nikhil-Ladha thanks for the guidance